src/reducers/settings.js

```diff
@@ -2,11 +2,22 @@
 
 export const ADD_SETTINGS = 'ADD_SETTINGS'
 
+const SYSTEM_SETTINGS_DIGITAL_GROUP_SEPARATOR = 'keyAnalysisDigitGroupSeparator'
+export const SYSTEM_SETTINGS_HIDE_DAILY_PERIODS = 'keyHideDailyPeriods'
+export const SYSTEM_SETTINGS_HIDE_WEEKLY_PERIODS = 'keyHideWeeklyPeriods'
+export const SYSTEM_SETTINGS_HIDE_BIWEEKLY_PERIODS = 'keyHideBiWeeklyPeriods'
+export const SYSTEM_SETTINGS_HIDE_MONTHLY_PERIODS = 'keyHideMonthlyPeriods'
+export const SYSTEM_SETTINGS_HIDE_BIMONTHLY_PERIODS = 'keyHideBiMonthlyPeriods'
+
 export const DEFAULT_SETTINGS = {
     // keyDateFormat: 'yyyy-MM-dd',
     // keyAnalysisRelativePeriod: 'LAST_12_MONTHS',
-    keyAnalysisDigitGroupSeparator: 'SPACE',
-    // keyIgnoreAnalyticsApprovalYearThreshold: -1,
+    [SYSTEM_SETTINGS_DIGITAL_GROUP_SEPARATOR]: 'SPACE',
+    [SYSTEM_SETTINGS_HIDE_DAILY_PERIODS]: false,
+    [SYSTEM_SETTINGS_HIDE_WEEKLY_PERIODS]: false,
+    [SYSTEM_SETTINGS_HIDE_BIWEEKLY_PERIODS]: false,
+    [SYSTEM_SETTINGS_HIDE_MONTHLY_PERIODS]: false,
+    [SYSTEM_SETTINGS_HIDE_BIMONTHLY_PERIODS]: false,
     displayNameProperty: 'displayName',
```

Comment:
Should this one be controlled by `keyAnalysisDisplayProperty`?

Reply:
Yes potentially. I'll look into it next, when I do the settings refactor